gdscript/sidebar.gd

```diff
@@ -337,6 +310,7 @@ func init(end_task: bool = true) -> void:
 	if not check_and_prompt_for_user_name(self._check_for_user_branch):
 		return
 
+	# todo: miserable UI that waits 5 seconds before nagging the user to make a branch, make better
```

Comment:
I think we should just remove this feature entirely. It adds more confusion than it helps